doc/ch-image.rst

```diff
@@ -927,6 +928,156 @@ trading off the time needed to retrieve an image from cache.
    Glob characters must be quoted or otherwise protected from the shell, which
    also desires to interpret them and will do so incorrectly.
 
+:code:`export`
+===============
+
+::
+
+   $ ch-image [...] export [-o FMT] [...] IMAGE_REF PATH
+
+Description
+-----------
+
+Copy image :code:`IMAGE_REF` to :code:`PATH` and convert its format. Replace
+:code:`PATH` if it already exists, unless :code:`--no-clobber` is specified. It
+is an error if :code:`IMAGE_REF` and :code:`PATH` have the same format; use the
+format's own tools for that case.
```

Comment:
```suggestion
Copy image :code:`IMAGE_REF` from builder storage to :code:`PATH` in format
:code:`FMT` if provided; otherwise, use default format :code:`tar`. Replace
image at :code:`PATH` if it already exists, unless :code:`--no-clobber` is
specified.
```